Gemfile

```diff
@@ -109,6 +109,9 @@ gem 'jquery-migrate-rails'
 gem 'jquery-rails', '4.4.0'
 gem 'jquery-ui-rails', '~> 4.2'
 gem 'select2-rails', '~> 3.4.7'
+# Thor v0.20 works with both select2-rails 3.4.7 and railties 6.0.3.6
+#   To upgrade to thor v1 we need to upgrade select2-rails to v3.5 which requires some work
```

Comment:
I think we should aim to completely ditch `select2`in the short-medium term and replace it with something more modern/lightweight/portable that doesn't depend directly on `jQuery`, like [SlimSelect](https://slimselectjs.com/).

It's not a job for today though...

Reply:
I added this point to this issue here: https://github.com/openfoodfoundation/openfoodnetwork/issues/6510